.erb-lint.yml

```diff
@@ -15,6 +15,8 @@ linters:
         Enabled: false
       Layout/TrailingWhitespace:
         Enabled: false
+      Layout/LeadingBlankLines:
+        Enabled: false
```

Comment:
I didn't manage to make this rule happy with my code on app/views/providers/check_benefits/index.html.erb line 5 to 20 (see below)